src/components/BulkEditActionMenu/BulkEditActionMenu.js

```diff
@@ -6,20 +6,40 @@ import {
   Button,
   Icon,
 } from '@folio/stripes/components';
+import { useDownloadErrors } from '../../API/useErrorDownload';
+import { useParams } from '../../hooks/useParams';
 
 export const BulkEditActionMenu = ({
   onEdit,
   onDelete,
   onToggle,
 }) => {
+  const { id } = useParams('/bulk-edit/:id');
+
   const buildButtonClickHandler = buttonClickHandler => () => {
     buttonClickHandler();
 
     onToggle();
   };
 
+  const { data } = useDownloadErrors(id);
```

Comment:
data?.files[1] - it's hard to read a little bit. May we do something like this?

`const { data: { files } } = useDownloadErrors(id);`
`const [successCsvLink, errorCsvLink] = files;`

Reply:
I thought about that. It will be great to do in this way, coz it will be easy to use in other stories related to download files. I will fix it right now.